app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -248,58 +249,40 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
     skip: !useIsAuthed(),
   })
 
-  const { moneyAmountToDisplayCurrencyString, formatMoneyAmount, displayCurrency } =
-    useDisplayCurrency()
+  const {
+    formatDisplayAndWalletAmount,
+    formatMoneyAmount,
+    getSecondaryAmountIfCurrencyIsDifferent,
+  } = useDisplayCurrency()
   const { LL } = useI18nContext()
-  const { convertMoneyAmount } = usePriceConversion()
+  const { convertMoneyAmount: _convertMoneyAmount } = usePriceConversion()
```

Comment:
why is there a need to rename to `_convertMoneyAmount`?

Reply:
To ensure the same price is being used everywhere, convertMoneyAmount should come from the paymentDetails object, so we use _convertMoneyAmount to be put into the paymentDetails object and then from the paymentDetails object we grab the function and name it convertMoneyAmount.